examples/fabric/v4/cloudRouterConnectivity/cloudRouter2wan/variables.tf

```diff
@@ -0,0 +1,18 @@
+variable "equinix_client_id" {}
+variable "equinix_client_secret" {}
+
+variable "fcr_uuid" {}
+variable "network_uuid" {}
+
+variable "notifications_type" {}
+variable "notifications_emails" {}
+variable "purchase_order_number" {}
+
+variable "connection_name" {}
+variable "connection_type" {}
```

Comment:
Should add a `terraform.tfvars.example` file with example values for the user to easily copy into a local `terraform.tfvars` file.